pkg/ai/iai.go

```diff
@@ -168,12 +171,12 @@ func (p *AIProvider) GetCompartmentId() string {
 	return p.CompartmentId
 }
 
+var passwordlessProviders = []string{"localai", "ollama", "amazonsagemaker", "amazonbedrock", "googlevertexai", "oci", "watsonxai","ollama"}
```

Comment:
need linting.

Reply:
Thanks @JuHyung-Son 

It has been changed :-)